play-json/jvm/src/main/scala/play/api/libs/json/PlatformReads.scala

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package play.api.libs.json
+import java.util.Locale
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.node.{ ArrayNode, ObjectNode }
+import play.api.libs.json.jackson.JacksonJson
+
+trait PlatformReads {
```

Comment:
I don't see the point of adding an extra level of inheritance. In master, the shared DefaultReads extends EnvReads for env specific bindings. For me these 2 levels are sufficient.

Reply:
The extra level is required to allows to keep the JVM only serializers/Deserializers separated by the shared ones. I  refactored in common place a lot on serializers/deserializers used for java.time to be used both in JVM and JS

Reply:
For me the separation between shared/common and specific js/JVM is already there on master so still don't see the point